### ISSUE TITLE
Datatable [accessibility]: Narrator didn't announce  rowGroupMode="rowspan"

### ISSUE THREAD

**Issue by arnodemer** [primefaces/primereact#6948]
### Describe the bug

when a datatable uses rowGroupMode="rowspan"  (and  selectionMode="single"), narrator announce correctly grouped column value for first row, but not for other rows

### Reproducer

https://stackblitz.com/edit/y9p747?file=src%2FApp.jsx,src%2Fservice%2FCustomerService.jsx

### System Information

primereact 10.8.0

### Steps to reproduce the behavior

1. open reproducer
2. start NVDA narrator
3. click on "Take focus" button
4. hit tab to go through first then second table
5. "browser" is annouced only for first row of first table

### Expected behavior

grouped rows column value should be announced for every rows.
